### PR TITLE
make dialog examples more beautiful

### PR DESCRIPTION
**Pull request by DaNautilus**

### CODE REVIEW

File: src/scss/_dialog.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+ux-dialog-overlay {
```
Comment (dweber019) [inline src/scss/_dialog.scss]: Please comment this file that this changes are related to Bootstrap.

File: src/app/resources/elements/edit-person/edit-person.element.html [hunk 2]
```diff
@@ -1,13 +1,27 @@
 <template>
```
Comment (dweber019) [inline src/app/resources/elements/edit-person/edit-person.element.html]: Add a comment and link to the original implementation of the dialog html as this is a Bootstrap specific implementation for the dialog.

File: src/scss/_dialog.scss (hunk 1 above)
Comment (DaNautilus) [reply src/scss/_dialog.scss]: Done

File: src/app/resources/elements/edit-person/edit-person.element.html (hunk 2 above)
Comment (DaNautilus) [reply src/app/resources/elements/edit-person/edit-person.element.html]: Done

**Pull request merged**